pandas/core/arrays/interval.py

```diff
@@ -1518,7 +1518,11 @@ def delete(self: IntervalArrayT, loc) -> IntervalArrayT:
         return self._shallow_copy(left=new_left, right=new_right)
 
     @Appender(_extension_array_shared_docs["repeat"] % _shared_docs_kwargs)
-    def repeat(self: IntervalArrayT, repeats: int, axis=None) -> IntervalArrayT:
+    def repeat(
+        self: IntervalArrayT,
+        repeats: int | Sequence[int],
+        axis: int | None = None,
```

Comment:
can you just the Axis typing signature

Reply:
@jreback I could, but the docs say the following about `axis`:
```
        axis : None
            Must be ``None``. Has no effect but is accepted for compatibility
            with numpy.
```
And `Axis` is defined as ` Union[str, int]`, and the `str` wouldn't be compatible with numpy.

Reply:
Then let's define a new typing alias NumpyAxis or similar.

Reply:
could also make Axis more obvious with e.g. `Literal["index", "rows", "columns", 0, 1]`

Reply:
@jbrockmendel No, we can't accept those arguments in `ExtensionArray.repeat()`, because we call `numpy.repeat`